apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -2043,16 +2043,21 @@
       "SUCCESS": "Owners successfully removed"
     },
     "DELETE_ENTITY": {
-      "BASIC": "Do you want to delete",
+      "BASIC": "Do you want to {{type}}",
       "ONLY": "only this",
```

Comment:
Some of these translations could be merged, with the entity type being a parameter. Mainly "ONLY", "OR", "RELATIONS".

Reply:
nah, when you look at it more, it makes sense
the "or" is here specifically because we want to make it bold
the "relations" is here specially because is reused on more places